sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosException.java

```diff
@@ -37,6 +37,8 @@
 public class CosmosException extends AzureException {
     private static final long serialVersionUID = 1L;
 
+    private final static String SDK_VERSION = HttpConstants.Versions.SDK_VERSION;
```

Comment:
Is there any way to get the UserAgent? That way it will contain other information needed for troubleshooting?

Reply:
Agreed on putting the complete user agent.